script/testing/junit/src/TracefileTest.java

```diff
@@ -56,31 +47,30 @@ public void setUp() throws Throwable {
         Collection<DynamicTest> dTest = new ArrayList<>();
         int lineCounter = -1;
         // get all query start numbers
-        List<Integer> queryLine = null;
-        queryLine = getQueryLineNum(file);
+        List<Integer> queryLine = TestUtility.getQueryLineNum(file);
 
         // loop through every sql statement
         while (mog.next()) {
             // case for create and insert statements
             lineCounter++;
             String cur_sql = mog.sql.trim();
             int num = queryLine.get(lineCounter);
-            if (mog.queryResults.size() == 0) {
+            if (mog.queryFirstLine.equals(Constants.STATEMENT_ERROR)||mog.queryFirstLine.equals(Constants.STATEMENT_OK)) {
                 Statement statement = null;
                 Executable exec = null;
-                String testName = "Line:" + num + " | Expected " + mog.status;
+                String testName = "Line:" + num + " | Expected " + mog.queryFirstLine;
                 try {
                     statement = conn.createStatement();
                     statement.execute(cur_sql);
-                    if(mog.status.equals(ERROR)){
+                    if(mog.queryFirstLine.equals(Constants.STATEMENT_ERROR)){
                         String message = "Failure at Line " + num + ": Expected failure but success"  + "\n " + cur_sql;
                         exec = () -> check2(message);
                     }else{
                         exec = () -> assertEquals(true, true);
                     }
                 }
                 catch (Throwable e) {
-                    if(mog.status.equals(OK)){
+                    if(mog.queryFirstLine.equals(Constants.STATEMENT_OK)){
                         String message = "Failure at Line " + num + ": Expected success but failure"  + "\n " + cur_sql;
                         exec = () -> check2(message);
                     }else{
```

Comment:
The `generateTest` function is getting too large. Split out the two main if-clause branches into separate functions: `executeQueryNoResults` and `executeQueryWithResults`